bigquery/google/cloud/bigquery/table.py

```diff
@@ -906,6 +906,42 @@ def __init__(self, resource):
 
         self._properties = resource
 
+    @property
+    def created(self):
+        """Union[datetime.datetime, None]: Datetime at which the table was
+        created (:data:`None` until set from the server).
+        """
+        creation_time = self._properties.get("creationTime")
+        if creation_time is not None:
+            # creation_time will be in milliseconds.
+            return google.cloud._helpers._datetime_from_microseconds(
+                1000.0 * float(creation_time)
+            )
+
+    @property
+    def expires(self):
+        """Union[datetime.datetime, None]: Datetime at which the table will be
+        deleted.
+
+        Raises:
```

Comment:
The `Raises` section can be removed when the setter is removed.

Reply:
Removed the raises comment since setter was removed!